MaterialLibrary/src/main/java/io/doist/material/widget/utils/MaterialWidgetHandler.java

```diff
@@ -256,4 +350,21 @@ private static void initTextViewAttributes(Context context, MaterialResources re
         }
         return newStyleable;
     }
+
+    private static int getThemeResId(Context context) {
```

Comment:
We can probably turn `MaterialResources$getThemeResId(Context)` into a public method. What do you think?